### DISCUSSION TITLE
Tasmota and L298N possible?

### DISCUSSION THREAD

**Discussion by tung256** [arendst/Tasmota#18158] [Q&A] [answered]
has anyone gotten this L298N+Nema 17 to work with ESPHome or Tasmota?  if so, could you please show the wiring diagram as well as the codes?
i see people doing L298N driver for Nema17 stepper motor with Arduino so i think it must be possible with esp8266 loaded with esphome or Tasmota right?

**Comment by tung256** [comment, marked as answer]
this is the wiring diagram i found.  no idea how the code nor config looks like with Tasmota
![image](https://user-images.githubusercontent.com/37672036/224445972-8f6a9d36-dab4-409c-b141-4cb8baf181f3.png)

**Reply by Jason2866** [reply]
Not supported. Tasmota supports stepper driver A4988

**Reply by tung256** [reply]
is there any method to get L298N and Nema17 into HomeAssistant then?  i searched for Esphome and no luck as well. all this time i thought if Arduino can do it then Tasmota or Esphome can  :(

**Reply by barbudor** [reply]
If you know how to do it in Arduino, you may contribute to Tasmota by adding such driver
